src/Uno.UI.Runtime.Skia.X11/X11XamlRootHost.x11events.cs

```diff
@@ -71,9 +71,20 @@ private unsafe void Run()
 
 		while (true)
 		{
-			var ret = X11Helper.poll(fds, 1, -1); // infinite waiting
+			var ret = X11Helper.poll(fds, 1, 1000);
```

Comment:
Why 1000? Can you add a comment explaining it?

Reply:
Just a random timeout to check if the window is closed (so that we don't wait forever), then we go back to polling again if the window is still up. I will add a comment